### PR TITLE
[18.0][MIG] account_move_line_stock_info

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by JordiMForgeFlow — APPROVED** [review]
code review 👍🏼

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py [hunk 1]
```diff
@@ -80,7 +80,6 @@ def _create_account(self, acc_type, name, code, company):
                 "name": name,
                 "code": code,
                 "account_type": acc_type,
-                "company_id": company.id,
```
Comment (MiquelRForgeFlow) [inline account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: use `company_ids` now

File: account_move_line_stock_info/models/account_move_line.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    stock_move_id = fields.Many2one(
+        comodel_name="stock.move", string="Stock Move", copy=False, index=True
```
Comment (AaronHForgeFlow) [inline account_move_line_stock_info/models/account_move_line.py]: Pplease use the same index as in https://github.com/odoo/odoo/tree/18.0/addons/stock_account/models#L10

Is this redefinition even needed?

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py (hunk 1 above)
Comment (JasminSForgeFlow) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: Updated, Thanks

File: account_move_line_stock_info/models/account_move_line.py (hunk 2 above)
Comment (JasminSForgeFlow) [reply account_move_line_stock_info/models/account_move_line.py]: index updated,
and field is availble on account.move not on account.move.line

Thanks

Comment (AaronHForgeFlow) [reply account_move_line_stock_info/models/account_move_line.py]: Right. Sorry.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — APPROVED** [review]

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**